Tasks/VsTest/package-lock.json

```diff
@@ -63,7 +63,7 @@
     "minimatch": {
```

Comment:
do we need a package-lock.json file? I know it gets auto generated, but what is its purpose?